qemu/qmp_test.go

```diff
@@ -1342,7 +1342,7 @@ func TestExecMemdevAdd(t *testing.T) {
 	cfg := QMPConfig{Logger: qmpTestLogger{}}
 	q := startQMPLoop(buf, cfg, connectedCh, disconnectedCh)
 	checkVersion(t, connectedCh)
-	err := q.ExecMemdevAdd(context.Background(), "memory-backend-ram", "mem0", "", 128, true, "virtio-mem-pci", "virtiomem0")
+	err := q.ExecMemdevAdd(context.Background(), "memory-backend-ram", "mem0", "", 128, true, "virtio-mem-pci", "virtiomem0", "", "")
```

Comment:
I'd suggest testing non-empty bus and addr here, since virtio-mem is a case that's very likely to need this.

Reply:
@dgibson thanks, I'll check if is possible to use bridges in tests an re-spin